app/controllers/followees_controller.rb

```diff
@@ -4,6 +4,7 @@ class FolloweesController < ApplicationController
 
   def index
     @users = @user.followees(User)
```

Comment:
Consider renaming `@users` to a more specific name that explains who these `@users` are.